src/components/SideMenu.tsx

```diff
@@ -15,16 +15,15 @@ const SIDE_MENU_STORAGE_KEY = "SIDEMENU";
 const components: Array<ComponentsListItem> = comps.sort((a, b) => (a.name > b.name ? 1 : -1));
 
 function getStoredValue(): boolean {
-    const value: string = localStorage.getItem(SIDE_MENU_STORAGE_KEY);
+    const w: Window | null = typeof window !== "undefined" ? window : null;
+    const value: string = w === null ? "true" : w.localStorage.getItem(SIDE_MENU_STORAGE_KEY);
```

Comment:
I don't like this solution 😐

Reply:
I understand the need for it, but I feel like the [suggested fix from Gatsby](https://www.gatsbyjs.com/docs/debugging-html-builds/#how-to-check-if-window-is-defined) is way cleaner

Reply:
Check #475 for reference